client/controllers/incidentReports.coffee

```diff
@@ -157,15 +157,14 @@ Template.incidentReports.helpers
         key: 'dateRange'
         label: 'Date'
         fn: (value, object, key) ->
-          dateFormat = 'M/D/YYYY'
+          dateFormat = 'MMM D, YYYY'
           if object.dateRange?.type is 'day'
             if object.dateRange.cumulative
               return "Before " + moment(object.dateRange.end).format(dateFormat)
             else
               return moment(object.dateRange.start).format(dateFormat)
           else if object.dateRange?.type is 'precise'
-            return moment(object.dateRange.start).format(dateFormat) + ' - '
-            + moment(object.dateRange.end).format(dateFormat)
+            return moment(object.dateRange.start).format(dateFormat) + ' - ' + moment(object.dateRange.end).format(dateFormat)
```

Comment:
@nathanathan for some reason I was getting weird output formatting when I left the line break.  It was showing `1/1/2016 -` instead of `1/1/2016 - 1/2/2016`

If you think it's important I can go back and fiddle with it?